index.ts

```diff
@@ -104,8 +104,19 @@ export {
     springBootGradleArgs,
     ListGradleBranchDeploys,
 } from "./lib/gradle/deploy/GradlePerBranchSpringBootDeploymentGoal";
-
 export * from "./lib/riff/riffGeneration";
 export * from "./lib/riff/riffPushTests";
-
 export * from "./lib/riff/RiffDeployment";
+export {
+    addPluginTransform,
+    addManagedPluginTransform,
+    AddManagedMavenPlugin,
+    AddMavenPlugin,
+} from "./lib/maven/transform/addPluginTransform";
+export {
+    ManagedPlugin,
+    Plugin,
+    PluginExecution,
+    PluginExecutionGoal,
+    Plugins,
+} from "./lib/maven/Plugin";
```

Comment:
this is not essential
but you might consider exporting these in an explicit `maven` namespace, because they are general terms. We really mean maven plugin. We could call them MavenPlugin, or maven.Plugin

that's totally a matter of opinion though, people can import them however they choose, namespacing them here would only change the easiest way of importing them.